fullstack/server/.env

```diff
@@ -0,0 +1,4 @@
+AWS_ACCESS_KEY_ID=AKIAYWSE7UDGYKN3D6T4
+AWS_SECRET_ACCESS_KEY=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
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Aws Secret Access Key' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 334256ee-c7af-4a92-bea9-fb53e0d5404c -->